evident/tests/test-effect-size.py

```diff
@@ -319,6 +324,19 @@ def test_effect_size_alpha(self):
                     results_site_site['pooled_pval'],
                     results_race_race['pooled_pval'])
 
+        # check summary results
+        sn = pd.read_csv(join(output, 'alpha_sn.txt.tsv'),
+                         sep='\t', index_col=0)
+        otu = pd.read_csv(join(output, 'alpha_otu.txt.tsv'),
+                          sep='\t', index_col=0)
```

Comment:
@serenejiang, thank you for adding these tests. Could you add a comment block briefly explaining the idea of the tests and why using assert_array_less works for this?

Reply:
added comments of tests in the latest pull request